tests/test_template.py

```diff
@@ -37,6 +38,18 @@ def test_transform(self):
         template.set_transform(transform)
         self.assertEqual(template.transform, transform)
 
+    def test_globals(self):
+        template = Template()
+        globals = Globals()
+        with self.assertRaises(ValueError):
+            template.set_globals(globals)
+        transform = "AWS::Serverless-2016-10-31"
```

Comment:
Was this intentional to be an explicit string rather than importing SERVERLESS_TRANSFORM?